ubuntu.md

```diff
@@ -0,0 +1,31 @@
+### Ubuntu 16.04
+Instructions for installing VirtualBox & Vagrant on Ubuntu 16.04
+
+```shell
+sudo apt update
+sudo apt -y --force-yes upgrade
+sudo reboot
+```
+
+```shell
```

Comment:
more comment is (almost) always better. Please push to the branch `update_readme`